index.bs

```diff
@@ -133,9 +138,9 @@ div.enum-description > table > thead > tr > th {
 
         [Exposed=Window]
         partial interface Navigator {
-          AutoplayPolicy getAutoplayPolicy(HTMLMediaElementConstructor ctor);
+          AutoplayPolicy getAutoplayPolicy(HTMLMediaElementConstructor mediaCtor);
```

Comment:
Should these renaming changes belong to #20, not this one?

Reply:
Yes, I had integrated the change, because build with Bikeshed (using default settings) otherwise fails. In other words, the workflow will fail until #20 is merged (which is OK, I guess).